instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -68,6 +73,31 @@ public SdkTracerProvider sdkTracerProvider(
           .build();
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public SdkLoggerProvider sdkLoggerProvider(
+        ObjectProvider<List<LogRecordExporter>> loggerExportersProvider, Resource otelResource) {
+
+      SdkLoggerProviderBuilder loggerProviderBuilder = SdkLoggerProvider.builder();
+      loggerProviderBuilder.setResource(otelResource);
+
+      initializeGlobalLoggerProvider(loggerExportersProvider);
+
+      return loggerProviderBuilder.build();
+    }
+
+    private static void initializeGlobalLoggerProvider(
+        ObjectProvider<List<LogRecordExporter>> loggerExportersProvider) {
+      SdkLoggerProviderBuilder loggerProviderBuilder = SdkLoggerProvider.builder();
+      loggerExportersProvider.getIfAvailable(Collections::emptyList).stream()
+          .forEach(
+              loggerExporter ->
+                  loggerProviderBuilder.addLogRecordProcessor(
+                      SimpleLogRecordProcessor.create(loggerExporter)));
+      SdkLoggerProvider loggerProvider = loggerProviderBuilder.build();
```

Comment:
You're building the logger provider twice: here and in line 86. Instead of having this method, WDYT about building the `SdkLoggerProvider` the same way `SdkTracerProvider` and `SdkMeterProvider` (+ extra consistency), but saving the result of `build()` to a local var before returning it?